lib/Open/index.js

```diff
@@ -1,6 +1,7 @@
 const fs = require('graceful-fs');
 const directory = require('./directory');
 const Stream = require('stream');
+const { GetObjectCommand, HeadObjectCommand } = require('@aws-sdk/client-s3');
```

Comment:
This will not work, as this will throw an error if aws-sdk is not installed, whether you are using the s3_v3 function or not.   That's not what we want.

Moving the require inside the s3_v3 would work though, as an error would only occur when s3_v3 is explicitly called and @aws-sdk/client-s3 is not installed.

Reply:
Import moved